### ISSUE TITLE
MH Report

### ISSUE THREAD

**Issue by jonathansberry** [meerkat-code/meerkat_frontend#260]
"To implement changes to mental health report"

**Comment by mixmixmix** [comment]
This is a duplicate of the issue #152. I'll close this once #152 is ready.

**Comment by jonathansberry** [comment]
Ah sure.  We'll just replace then.
